src/tests/Microsoft.PowerFx.Connectors.Tests/BasicRestTests.cs

```diff
@@ -137,5 +140,53 @@ public async Task PetStore_MultiServer()
             // Ensure we use HTTPS protocol
             Assert.Equal("https", apiDoc.GetScheme().Substring(0, 5));
         }
+
+        [Fact]
+        public async Task RealTest()
+        {
+            var config = new PowerFxConfig();
+
+            var swaggerUrl = "https://api.math.tools/yaml/math.tools.numbers.openapi.yaml";
+
+            // Other intersting files:
+            // "https://api.apis.guru/v2/specs/weatherbit.io/2.0.0/swagger.json"
+            // "https://www.weatherbit.io/static/swagger.json"
+
+            var doc = await ReadSwaggerFromUrl(swaggerUrl).ConfigureAwait(false);
+
+            using var client = new HttpClient(); // public auth 
```

Comment:
Note this goes to a public endpoint directly and **completely skips APIM.**  So no ConnectionId, and just uses a plain HttpClient, not a PowerPlatformHttpClient.